docs/extensions/accounts/tests.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Tests"
+weight: 80
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/extensions/accounts
+geekdocFilePath: tests.md
+---
+
+{{< toc >}}
+
+## Requirements
+
+You need a working installation of [the Go programming language](https://golang.org/), [the Node runtime](https://nodejs.org/) and [the Yarn package manager](https://yarnpkg.com/) installed to run the acceptance tests. You may also want to use [Docker](https://www.docker.com/) to start the necessary services in their respective containers.
+
+## Acceptance Tests
+
+Make sure you've cloned the [web frontend repo](https://github.com/owncloud/web/) and the [infinite scale repo](https://github.com/owncloud/ocis/) next to each other. If your file/folder structure is different, you'll have to change the paths below accordingly.
+
+### In the web repo
+
+Install dependencies and build the frontend with a watcher by running
+
+```
+yarn && yarn build:w
+```
+
+Also, start the necessary services. The easiest way to do this is using Docker (Compose):
+
+```
+docker compose up selenium middleware-ocis vnc ocis
```

Comment:
I usually run `docker compose up -d` to start all services. As oc10 is not relevant here your command is better / more precise. But mine is simpler. 😁 Maybe you can add a hint? Not needed, please feel free to ditch this comment.

Reply:
Ditched since starting all of them would clash with the binary we need ;)